client/src/components/list-container/list-item-component/__tests__/__snapshots__/ListItemComponent.test.jsx.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`render snapshots renders correctly 1`] = `<listItemComponent />`;
+
+exports[`render snapshots renders correctly with name 1`] = `
+<listItemComponent
```

Comment:
this is wha i talked about in the last comment.

snapshot should be the end result of the html, pure html tags, text, classes..